bridges/CarThrottleBridge.php

```diff
@@ -8,14 +8,62 @@ class CarThrottleBridge extends BridgeAbstract
     const MAINTAINER = 't0stiman';
     const DONATION_URI = 'https://ko-fi.com/tostiman';
 
+    const PARAMETERS = array(
```

Comment:
short array

Reply:
what do you mean?

Reply:
longform array is disallowed. shortform array is required.